.ddev/commands/web/check

```diff
@@ -30,6 +30,10 @@ echo "-------------"
 echo "Running PHPCS"
 echo "-------------"
 vendor/bin/phpcs web/modules/contrib/$1 --standard="Drupal,DrupalPractice" -n --extensions="php,module,inc,install,test,profile,theme"
+echo "----------------------------------------------------"
+echo "Running PHPCS with PHP Compatibility coding standard"
+echo "----------------------------------------------------"
+vendor/bin/phpcs -p web/modules/contrib/$1 --standard="PHPCompatibility" --runtime-set testVersion "8.1"
```

Comment:
Let's also create a phpversion command that lets you pass a testVersion var (and defaults to 8.1)